Automattic/wp-calypso
# Plans: Remove `plansFeatureList` A/B test

This removes the test from #2924 and reverts to showing the description for each plan. Further refinement of the test is required at this time.

**Testing**

* Visit http://calypso.localhost:3000/plans
* Select a site to view the plans
* Run `localStorage.setItem( 'ABTests', '{"plansFeatureList_20040202":"list"}' );` in your console
* Refresh and confirm you see the description text under the plans
